### ISSUE TITLE
Doesn't build under Ubuntu 

### ISSUE THREAD

**Issue by DevraRabbit** [chetant/LibClang#63]
I rebuild ghc, haskell-plattform and even LibClang from source but everytime it gives me the same error:

```
Setup.hs:17:8:
    Could not find module `Distribution.Simple.Program.Find'
    Perhaps you meant
      Distribution.Simple.Program.Ld (from Cabal-1.16.0)
      Distribution.Simple.Program.Run (from Cabal-1.16.0)
      Distribution.Simple.Program.Ar (from Cabal-1.16.0)
    Use -v to see a list of the files searched for.
```

Do I have to fix something in the dependencies?

Thanks

**Comment by TomMD** [comment]
The issue is the version of the cabal library. I don't know if hand if it is expecting something older or newer. Cabal update, cabal install Cabal cabal-install, retry?

**Comment by chetant** [comment]
The Cabal version should be >=1.18
Will update the Cabal file to match that version dependency

**Comment by chetant** [comment]
Fixed in e8723e4
